news/2020-03-22-monthly-beatmapping-contest-2-results.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "Monthly Beatmapping Contest #1: Results"
+date: 2020-03-22 09:00:00 +0000
+---
+
+February's Monthly Beatmapping Contest has come to a close. Find out who wears the crown!
+
+![](/wiki/shared/news/banners/monthly-beatmapping-contest.png)
+
+For those unfamiliar, the **Monthly Beatmapping Contest** (***MBC***) began its comeback in 2020. Everything you'd need to know about the contest is detailed on the [MBC wiki article](/wiki/Contests/Monthly_Beatmapping_Contest), so give it a read.
+
+## Winners
+
+Votes for all finalists can be found on the [contest's listing](https://osu.ppy.sh/community/contests/89). The main requirement for submissions was that **each could not exceed 4.00 stars.**
+
+**Community vote winner:** **[Voicians - Wolves](https://assets.ppy.sh/contests/89/entries/Voicians%20-%20Wolves%20(Conspicuous%20Kite).osz?2020)** by **[Aphestra](https://osu.ppy.sh/users/11949191)**, masked as Conspicuous Kite.
```

Comment:
I feel like you don't really need this and could include the relevant info in next paragraph instead, it reads awkward since this isn't a normal sentence